salt/orchestrate/pnda.sls

```diff
@@ -230,6 +230,14 @@ orchestrate-pnda-app-packages:
     - timeout: 120
     - queue: True
 
+orchestrate-pnda-app-packages-hdfs-sync:
```

Comment:
could you please include hdfs-sync as part of the init.sls instead of adding an extra step on orchestrate. now that we have default kafka & avro dependencies, should be done as part of standard app-packages installation.
and hdfs-sync could still be call as an ops command